compiler/src/iree/compiler/Dialect/TensorExt/IR/TensorExtOpFolders.cpp

```diff
@@ -5,11 +5,50 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 #include "iree/compiler/Dialect/TensorExt/IR/TensorExtOps.h"
+#include "iree/compiler/Utils/ShapeUtils.h"
 #include "mlir/Dialect/Arith/Utils/Utils.h"
 #include "mlir/Dialect/Tensor/IR/Tensor.h"
 
 namespace mlir::iree_compiler::IREE::TensorExt {
 
+//===----------------------------------------------------------------------===//
+// iree_tensor_ext.bitcast
+//===----------------------------------------------------------------------===//
+
+struct ReplaceIfTensorOperandEmpty : public OpRewritePattern<BitCastOp> {
```

Comment:
nit: I'd rename it as `ReplaceBitCastIfTensorOperandEmpty` because it is at global scope in the cpp file. There are other TensorExt ops.